workflow.go

```diff
@@ -10,7 +10,7 @@ type WorkflowDefinition struct {
 
 // StepDefinition structure to handle workflow steps
 type StepDefinition struct {
-	Target     string               `yaml:"target,omitempty" json:"target,omitempty"`
+	Target     PropertyAssignment   `yaml:"target,omitempty" json:"target,omitempty"`
```

Comment:
Has the spec been updated to define `Target` as a PropertyAssignment?
Can you provide a link to the update spec where this was changed?